src/sh/sdv-kanto-ctl

```diff
@@ -0,0 +1,422 @@
+			echo "	$0 --reload remove-registry --hostname <hostname>"
+			exit 1
+		fi
+	elif [ "$1" = "remove-all-registries" ]; then
+		COMMAND=$1
+		shift
+		while [ -n "$1" ]; do 
+			echo "Error: Unknown argument for command remove-all-registries: $1"
+			exit 1
+		done
+	elif [ "$1" = "list-registries" ]; then
+		COMMAND=$1
+		shift
+		while [ -n "$1" ]; do 
+			echo "Error: Unknown argument for command list-registries: $1"
+			exit 1
+		done
+	elif [ "$1" = "show-config" ]; then
+		COMMAND=$1
+		shift
+		while [ -n "$1" ]; do 
+			echo "Error: Unknown argument for command show-config: $1"
+			exit 1
+		done
+	elif [ "$1" = "set" ]; then
+		COMMAND=$1
+		CONFIG_KEY=$2
+		CONFIG_VALUE=$3
+		if [ "$#" -ne 3 ]; then
+			echo "ERROR: set command requires two arguments: <key> and <value>"
+			exit 1
+		else
+			shift 2
+		fi
+	elif [ "$1" = "--no-reload" ] || [ "$1" = "-r" ]; then
+		RELOAD=0
+	elif [ "$1" = "--verbose" ] || [ "$1" = "-v" ]; then
+		VERBOSE=1
+	elif [ "$1" = "--ansi" ]; then
+		# reset COL_XX for monochrome output
+		export COL_NC=""
+		export COL_RED=""
+		export COL_GREEN=""
+		export COL_YELLOW=""
+		export COL_BLUE=""
+		export COL_WHITE=""
+		export COL_GRAY=""
+		export COL_NC_BOLD=""
+	elif [ "$1" = "--help" ]; then
+		display_help
+	else
+		echo "ERROR: Unkown command or option: $1";
+		echo "Try $0 --help"
+		exit 1
+	fi
+	if [ "$#" -gt 0 ]; then
+		shift
+    fi
+done
+
+init_colors
+
+#########################################
+# Read Local / Global SDV configuration #
+#########################################
+
+if [ -f "$HOME/.config/sdv.conf" ]; then
+	. "$HOME/.config/sdv.conf"
+	[ "$VERBOSE" = "1" ] && echo "# Config loaded from: $HOME/.config/sdv.conf" && grep -v  '^#' $HOME/.config/sdv.conf | sort -u
+elif [ -f "/etc/sdv/sdv.conf" ]; then
+	. "/etc/sdv/sdv.conf"
+	[ "$VERBOSE" = "1" ] && echo "# Config loaded from: /etc/sdv/sdv.conf" && grep -v  '^#' /etc/sdv/sdv.conf | sort -u
+else
+	[ "$VERBOSE" = "1" ] && echo "# Config file not found!"
+fi
+
+######################################################
+#                        commands                    #
+######################################################
+
+# Fragment for a new container registry configuration:
+add_registry() {
+	echo "Adding container registry $REGISTRY_HOSTNAME"
+	TMPFILE=$(mktemp --tmpdir kanto-cm-config.XXXXXXX)
+	jq \
+	--arg REGISTRY_HOSTNAME "$REGISTRY_HOSTNAME" \
+	--arg REGISTRY_USERNAME "$REGISTRY_USERNAME" \
+	--arg REGISTRY_PASSWORD "$REGISTRY_PASSWORD" \
+	'.containers.registry_configurations += {
+         ($REGISTRY_HOSTNAME): {
+             "credentials": {
+                 "user_id": $REGISTRY_USERNAME,
+                 "password": $REGISTRY_PASSWORD
+             }
+         } }' < ${KANTO_CONFIG} > ${TMPFILE}
+	[ "$VERBOSE" = "1" ] && echo "Created temp file for modifications: ${TMPFILE}"
+
+	if [ -w ${KANTO_CONFIG} ]; then
+		if ! diff ${TMPFILE} ${KANTO_CONFIG} > /dev/null; then
```

Comment:
diff > 1 might mean an error, so better check the exit codes properly.

Reply:
There is no operator >, Operator will be -gt I think